src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/courtsentencing/CourtSentencingReconciliationService.kt

```diff
@@ -557,6 +557,11 @@ class CourtSentencingReconciliationService(
     }
     return differences
   }
+  private fun LocalDate.asValidDate(): LocalDate? = if (this.isAfter(LocalDate.parse("1920-01-01")).and(this.isBefore(LocalDate.now().plusDays(1)))) {
```

Comment:
very minor, but would better be called asVaildOffenceDate due to future date check